client/view/index/Index.html

```diff
@@ -135,5 +209,44 @@ <h2>Try an example deck</h2>
 	components: {
 		SpreadsheetInput,
 	},
-}
+	methods: {
+		prepareDeleteSeenWorkbook: function(element) {
+			var sure = document.querySelector(".are-you-sure-window");
```

Comment:
It's relatively rare that you'll need to interact with an element directly, but when you do, you can use Svelte [refs](https://svelte.technology/guide#refs) instead of `querySelector`.